### ISSUE TITLE
Abaqus forces use of latest Intel Compiler

### ISSUE THREAD

**Issue by mikecroucher** [rcgsheffield/iceberg_software#59]
While writing the new user documentation for Abaqus, I noted an anomaly in that even though I had selected Intel Compiler 12.1 in my job submission file, it compiled using Intel 14.0
Investigation determined that we are doing something non-standard in our use of the ``abaqus`` command. Namely ::

    #!/bin/bash
    ## 
    #Record the use of this script by Stuart Mumford.
    # logger -p local1.debug "iceberg_software: USER=$USER TYPE=monitor SOFTWARE=/usr/local/bin/abaqus"
    #DS straight link will not work 
    module load compilers/intel

    export XLIB_SKIP_ARGB_VISUALS=1
    mesa=" "
    if [ "$1" == "cae" ] ;then
        mesa="-mesa"
    fi
    if [ -z $ABAQUSCOMMAND ] ; then 
       echo " Please use 'module load apps/abaqus' command first "
       exit 1
    fi
 
     $ABAQUSCOMMAND $* $mesa

My main issue with this is that the abaqus command will always load the latest version of the Intel compiler. The practical upshot of this is that if we add a new version of the Intel Compiler, we will be changing the abaqus environment for users without them necessarily knowing! We have needlessly coupled Intel Compiler to Abaqus! Such tight coupling is rarely a good idea 

This could lead to issues with reproducibility. 

In summary:

* It is impossible for us to upgrade the Intel Compiler without running the risk of breaking this Abaqus command. 
* We could inadvertently ruin a user’s set of results. E.g. if we update the Intel Compiler during a parameter sweep, different versions of the compiler will be used for different runs. This a Bad Idea!
*A user from a different site will expect to control which version of the Intel Compiler they use. Here, we always force the latest.
* This wrapper to abaqus is non-standard and may lead to further problems. It forces the -mesa switch for example. This switch is often, but not always, useful.

I suggest a change where we move a more standard way of working. Failing that, we should fully document this non-standard modification and warn users of the implications.

For now, I have modified the documentation such that it explicitly references the ``abq133`` command.

**Comment by Cadair** [comment]
I think that once we remove the access to abaqus in the default path, we should remove the load command from this script. Once we are requiring the user to run one module load command to run abaqus, needing them to manually load a compiler is the only logical approach.

**Comment by Cadair** [comment]
Also, we should not have our script stepping on the name of a default script. This will make it opaque to someone trying to use it, that we are doing something different, we should call it `runabaqus` or `ibabaqus` or something else.

**Comment by willfurnass** [comment]
@trallard a good time to look at this issue again if upgrading Abaqus?

**Comment by trallard** [comment]
@willfurnass good idea, we might also need to update the Intel Compilers as well as Abaqus though

**Comment by willfurnass** [comment]
@trallard Intel 15.0.3 and 17.0.0 on both Iceberg and Sharc - sufficient?

**Comment by trallard** [comment]
@willfurnass  need 16.1

**Comment by willfurnass** [comment]
@trallard Ah.  I think only @mikecroucher is able to download Parallel Studio products from Intel.

**Comment by trallard** [comment]
@willfurnass Nope, I made them accessible for CiCS already, you should be able to download them via the CiCS software download system now
